Gemfile

```diff
@@ -103,13 +101,23 @@ gem 'dry-types', '~> 1.7'
 # virtus-like DSL for defining typed struct classes.
 gem 'dry-struct', '~> 1.6'
 
+# Ruby gem for colorizing text using ANSI escape sequences.
+# Extends String class or add a ColorizedString with methods
+# to set the text color, background color and text effects.
+gem 'colorize'
+
 group :development, :test do
-  # See https://guides.rubyonrails.org/debugging_rails_applications.html#debugging-with-the-debug-gem
-  gem "debug", platforms: %i[ mri mingw x64_mingw ]
+  # This library provides debugging functionality to Ruby (MRI) 2.6
+  # and later.
+  # This debug.rb is replacement of traditional lib/debug.rb
+  # standard library which is implemented by set_trace_func. New
+  #debug.rb has several advantages:
+  # See: https://guides.rubyonrails.org/debugging_rails_applications.html#debugging-with-the-debug-gem
+  gem "debug", ">= 1.0.0"
 end
 
 group :development do
   # Speed up commands on slow machines / big apps [https://github.com/rails/spring]
   # gem "spring"
-  gem 'annotate'
+  gem 'annotate', git: 'https://github.com/ctran/annotate_models.git'
```

Comment:
The line gem 'annotate', git: 'https://github.com/ctran/annotate_models.git' in your Gemfile indicates that you want to include the annotate_models gem from the specified GitHub repository